types/types.pb.go

```diff
@@ -43,6 +44,7 @@ It has these top-level messages:
 //nolint: gas
```

Comment:
this gets removed if I run `make protoc`. Believe it or not, this is the one we need (right above `package` statement), not the one at the very top!